resources/devices/mobile-phone/htc.json

```diff
@@ -584,19 +597,6 @@
             "isTablet": "false"
         }
     },
-    "HTC Desire": {
```

Comment:
I brought this one into the "HTC A8181" device, which is also the HTC Desire, but with a code name.